From 5d3d683fd54100fea2a355b0ebc8b03d9b6c964b Mon Sep 17 00:00:00 2001
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Wed, 25 Jan 2023 13:57:09 +0100
Subject: [PATCH] kernel-install: remove math slang from man page

(cherry picked from commit 642617f431457382ec2140a52ee08bfbb3e5e1db)

Related: RHEL-16354
---
 man/kernel-install.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/man/kernel-install.xml b/man/kernel-install.xml
index b1822a8847..1bc6e7aa05 100644
--- a/man/kernel-install.xml
+++ b/man/kernel-install.xml
@@ -111,7 +111,7 @@
 
             <listitem><para><filename>90-uki-copy.install</filename> copies a file
             <filename>uki.efi</filename> from <varname>$KERNEL_INSTALL_STAGING_AREA</varname> or if it does
-            not exist the <replaceable>KERNEL-IMAGE</replaceable> argument, iff it has a
+            not exist the <replaceable>KERNEL-IMAGE</replaceable> argument, only if it has a
             <literal>.efi</literal> extension, to
             <filename>$BOOT/EFI/Linux/<replaceable>ENTRY-TOKEN</replaceable>-<replaceable>KERNEL-VERSION</replaceable>.efi</filename>.</para>
 
